keyboard:
  ros__parameters:
    joint_names: ['catheter_lin', 'catheter_rot', 'catheter_bend', 
      'sheath_lin', 'sheath_rot', 'sheath_bend']
    key_bindings:
      i: [1.0, 0.0, 0.0, 0.0, 0.0, 0.0]
      k: [-1.0, 0.0, 0.0, 0.0, 0.0, 0.0]
      l: [0.0, 1.0, 0.0, 0.0, 0.0, 0.0]
      j: [0.0, -1.0, 0.0, 0.0, 0.0, 0.0]
      o: [0.0, 0.0, 1.0, 0.0, 0.0, 0.0]
      u: [0.0, 0.0, -1.0, 0.0, 0.0, 0.0]
      w: [0.0, 0.0, 0.0, 1.0, 0.0, 0.0]
      s: [0.0, 0.0, 0.0, -1.0, 0.0, 0.0]
      d: [0.0, 0.0, 0.0, 0.0, 1.0, 0.0]
      a: [0.0, 0.0, 0.0, 0.0, -1.0, 0.0]
      e: [0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
      q: [0.0, 0.0, 0.0, 0.0, 0.0, -1.0]
    joint_vels: [5.0, 30.0, 30.0, 5.0, 30.0, 1.0]